build/common/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -186,6 +191,32 @@ def populateSettingValuesFromConfigMap(parsedConfig)
           puts "Using config map value: NODES_EMIT_STREAM_BATCH_SIZE = #{@nodesEmitStreamBatchSize}"
         end
       end
+
+      # networkflow logs settings
+      networkflow_logs_config = parsedConfig[:agent_settings][:networkflow_logs_config]
+      if !networkflow_logs_config.nil?
+        netoworkFlowLogsThrottleRate = networkflow_logs_config[:throttle_rate]
+        if is_valid_number?(netoworkFlowLogsThrottleRate)
+          @netoworkFlowLogsThrottleRate = netoworkFlowLogsThrottleRate.to_i
```

Comment:
we would need to telemetry related to throttle enabled or not and configuration.

Reply:
added